config/database.yml

```diff
@@ -8,6 +8,8 @@ defaults: &defaults
   pool: 5
   timeout: 5000
   host: localhost
+  variables:
+    statement_timeout: 250
```

Comment:
I'm just reviewing this for the first time. Have you increased this timeout value since making this comment?